src/components/FileViewer/FileContextMenu.tsx

```diff
@@ -90,9 +96,20 @@ const FileContextMenu = forwardRef(
             // subLabel={ isMac ? "⌘+X" : "CTRL+X"}
             onClick={() => {
               cutFile();
+              setShowContextMenu(false);
             }}
             disabled={file === null}
           />
+          <ContextMenuButton
+            className="w-full whitespace-nowrap rounded-none bg-gray-900 px-2.5 text-small font-medium"
+            label="Paste"
+            // subLabel={ isMac ? "⌘+C" : "CTRL+C"}
+            onClick={() => {
+              pasteFiles(file === null  || (file as ClientFile).file_type !== 'Directory' ? currentPath : (file as ClientFile).path);
```

Comment:
It might be better to add a bracket around `file === null || (file as ClientFile).file_type !== 'Directory'`.
I'm not too sure if `||` takes precedence over `? :`

Reply:
good call, added